db/migrate/20150202154317_create_join_table_activity_user.rb

```diff
@@ -0,0 +1,8 @@
+class CreateJoinTableActivityUser < ActiveRecord::Migration
+  def change
+    create_join_table :activities, :users do |t|
+       t.index [:activity_id, :user_id]
+       t.index [:user_id, :activity_id]
```

Comment:
This is what is generated automatically, i thought it would be neccasary somehow :)